src/lib/lightning/eclair/types.ts

```diff
@@ -54,7 +80,7 @@ export interface OpenChannelRequest {
   fundingSatoshis: number;
   pushMsat?: number;
   fundingFeerateSatByte?: number;
-  channelFlags?: number;
+  channelFlags?: ChannelFlags;
```

Comment:
Updated the type to a more specific one as the only possible values here are `0` and `1`.